src/desktop/apps/home/client/index.coffee

```diff
@@ -28,6 +28,20 @@ module.exports.HomeView = class HomeView extends Backbone.View
     new HomeAuthRouter
     Backbone.history.start pushState: true
 
+    targetElement = if sd.HOMEPAGE_COLLECTION_HUB_ENTRYPOINTS_TEST_QA is "experiment" then ".home-hubs-entry" else ".home-browse-module"
+    # .home-browse-module
+
+    # Remove after closing the homepage hubs entry points test
+    $(targetElement).waypoint(() ->
+      # Fire experiment viewed event
+      splitTest("homepage_collection_hub_entrypoints_test_qa").view()
+    ,
+    {
+      triggerOnce: true,
+      offset: 75,
+    }
+    )
```

Comment:
For context `waypoint` is [setup globally](https://github.com/artsy/force/blob/master/src/desktop/lib/global_client_setup.tsx#L90).